setup.sh

```diff
@@ -10,6 +10,10 @@ echo "Creating virtual environment..."
 virtualenv -p $python venv
 source venv/bin/activate
 echo "Installing dependencies..."
-pip install -r requirements.txt
```

Comment:
Thanks for the tip and sorry, i somehow missed the changes to requirements.txt. Should have been more careful when reading the changes.